csharp/private/actions/assembly.bzl

```diff
@@ -183,6 +183,7 @@ def AssemblyAction(
         out = out_file,
         refout = refout,
         pdb = pdb,
+        native_dlls = None,
```

Comment:
Oh, this is obviously wrong (as evidenced by CI). I just got as for as getting `bazel build :net20` in `every_framework_empty` happy